src/Equinox.Core/Cache.fs

```diff
@@ -101,6 +101,8 @@ type Cache private (inner: System.Runtime.Caching.MemoryCache) =
     member val Inner = inner
 
 type [<NoComparison; NoEquality; RequireQualifiedAccess>] CachingStrategy =
+    /// Do not apply any caching strategy for this Stream.
+    | NoCaching
```

Comment:
aka Kimi Raikkonen mode

Reply:
"The Iceman needs no cache"

Reply:
Indeed. (FWIW ref in my ind was "Leave me alone, I know what I'm doing").
`CachingStrategy.Kimi`